frontend/App.tsx

```diff
@@ -1,78 +1,134 @@
+  HomeScreen({ navigation }) {
+    console.log("homescreen")
+      return (
+        <View>
+          <View style={styles.container}>
+            <Text style={[Regstyles.text,{fontSize: 30}]}>Home Screen</Text>
+            <View style={Regstyles.row}>
+              <View style={{padding: 10}}>
+                <Button
+                  title="SearchContacts"
+                  onPress={() => navigation.navigate('SearchContacts')}
+                />
+              </View>
+              <View style={{padding: 10}}>
+                <Button
+                  title="Chat with me"
+                  onPress={() => navigation.navigate('ChatPanel')} 
+                />
+              </View>
+            </View>
+          </View>
+          <View>
+            {/* <Pressable 
+              style={Regstyles.btn} 
+              onPress={() => removeData(STORAGE_KEY)}>
+              <MaterialIcons name="delete" size={25}/>
+            </Pressable> */}
+          </View> 
+          {/* Trigger to call clearstorage function (function is not working) 
+          Error: cannot read properties of undefined */}
+        </View>
+      );
+  }
 
-  return (
-    <NavigationContainer>
-      <Stack.Navigator>
-        <Stack.Screen name="Home" component={HomeScreen} />
-        <Stack.Screen name="SearchContacts" component={SearchContacts} />
-        <Stack.Screen name="ChatPanel" component={ChatPanel} />
-      </Stack.Navigator>
-    </NavigationContainer>
-  );
+  DefaultScreen({ navigation }){
+    return(
+      <View>
+        <View>
+          <Text style={[Regstyles.text,{fontSize: 30}]}>HiChat</Text>
+          <Text style={[Regstyles.text,{fontSize: 20}]}>Connecting People...</Text>
+        </View>
+        <View>
+          <Pressable 
+            style={[Regstyles.button,{backgroundColor: 'black'}]} 
+            onPress={() => navigation.navigate('Registration')}>
+            <Text style={[Regstyles.text,{fontSize: 16, color: 'white',}]}>Sign Up</Text>
+          </Pressable> 
+        </View>
+      </View>
+    )
+  }
+  
+  render(){
+    console.log(this.state.phonenumber)
+    
+      return (
+        <>
+        { this.state.phonenumber ?
+          <NavigationContainer>
+            <Stack.Navigator>
+              <Stack.Screen name="HomeScreen" component={this.HomeScreen} />
+              <Stack.Screen name="SearchContacts" component={SearchContacts} />
+              <Stack.Screen name="ChatPanel" component={ChatPanel} />
+            </Stack.Navigator>
+          </NavigationContainer>
+        :
+          <NavigationContainer>
+            <Stack.Navigator>
+              <Stack.Screen name="DefaultScreen" component={this.DefaultScreen} />
+              <Stack.Screen name="Registration" component={Registration} />
+            </Stack.Navigator>
+          </NavigationContainer>
+       }
+        </>
+      );
+  }
 }
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
```

Comment:
Why remove flex?

Reply:
when used flex. The mobile view is getting upscaled and the elements in it are hidden. I don't know why it is happening